video/out/opengl/common.c

```diff
@@ -37,12 +37,10 @@
 static bool is_software_gl(GL *gl)
 {
     const char *renderer = gl->GetString(GL_RENDERER);
-    const char *vendor = gl->GetString(GL_VENDOR);
-    return !(renderer && vendor) ||
```

Comment:
Why is this line removed? `return !(renderer && vendor) ||`

As far as I can tell it's a safety measure, in case `gl->GetString(GL_RENDERER)` and/or `gl->GetString(GL_VENDOR)` is NULL.

If one of them does end up NULL, then without this line - the `strcmp` below can crash.